src/elements/content-sidebar/activity-feed/task-form/TaskForm.scss

```diff
@@ -25,10 +25,16 @@ $assignee-dropdown-height: 215px;
         min-width: 0;
 
         textarea,
-        input {
+        .date-picker-input {
             width: 100%;
         }
 
+        .checkbox-container {
+            // default margin is 20px between form input elements
+            // this checkbox needs to be 10px below the pill selector input
```

Comment:
Can we reduce the bottom margin of the pill selector rather than using negative margins?

Reply:
This is a conditionally rendered component, so reducing bottom margin of the pill selector will not make the spacing event between all the other input components in the form.